### PR TITLE
Docs: Detter documentation for Handles Pallet

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to make it easier to understand the normalization and validation for handles

### CODE REVIEW

**Review by enddynayn — APPROVED** [review]
👍 great!

File: pallets/handles/README.md [hunk 1]
```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
 Provides MSAs with an optional, but unique handle.
 
 A handle consists of:
-- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [UTF-8 Support](#utf-8-support) and [Homoglyph Attack Resistence](#homoglyph-attack-resistence) below.
+- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [Normalization Details](#normalization-details) below.
 - **Suffix:** A suffix is a unique numeric value appended to a user's base handle to make it unique.
```
Comment (jeanettedepatie) [inline pallets/handles/README.md:11]: "The suffix is a numeric value appended to the user's base handle to ensure the display handle (base handle + suffix) is unique."

File: pallets/handles/README.md [hunk 2]
```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
...
 
-### UTF-8 Support
+### Suffixes
 
-Handles are able to have many allowed ranges of UTF-8.
-Some ranges, such as emoji, are currently not allowed.
-Due to the handling of homoglyphs, some handles will resolve to the same value.
-For example, while the display may have diacriticals or homoglyphs, the handle is stored with diacriticals and homoglyphs normalized.
-So `Zoë.35` and `Zoe.35` will both resolve to the same MSA Id.
+TO allow multiple users to select the same handle, a distinguishing unique suffix is appended to to form the Display Handle.
```
Comment (jeanettedepatie) [inline pallets/handles/README.md:16]: In order to allow multiple users to select the same Base Handle, a unique numeric Suffix is appended to the Base Handle to form the Display Handle.

File: pallets/handles/README.md [hunk 3]
```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
...
+The suffix is generated from a random sequence such that each suffix is unique based on the normalized version of the handle.
+Thus for example, if there are two users who choose the handle `alice`, one would be `alice.57` and the other `alice.84`.
```
Comment (jeanettedepatie) [inline pallets/handles/README.md:18]: Remove thus or put a comma after it.

File: pallets/handles/README.md [hunk 4]
```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
...
 A handle consists of:
-- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [UTF-8 Support](#utf-8-support) and [Homoglyph Attack Resistence](#homoglyph-attack-resistence) below.
+- **Base Handle:** The user's chosen handle. It is *not* guaranteed to be unique without the suffix. It is linked to a normalized version for Handle to MSA Id resolution. See [Normalization Details](#normalization-details) below.
 - **Suffix:** A suffix is a unique numeric value appended to a user's base handle to make it unique.
 - **Display Handle:** The user's original (un-normalized, but with whitespace trimmed and consolidated) base handle string and the suffix together (`base`.`suffix`) constitute a unique identifier for a user.
 
-### UTF-8 Support
+### Suffixes
 
-Handles are able to have many allowed ranges of UTF-8.
```
Comment (jeanettedepatie) [inline pallets/handles/README.md:16]: Handles may...

File: pallets/handles/README.md [hunk 5]
```diff
@@ -7,25 +7,56 @@ Creates human-readable, homoglyph-attack resistant handles for MSAs.
...
-Some ranges, such as emoji, are currently not allowed.
-Due to the handling of homoglyphs, some handles will resolve to the same value.
-For example, while the display may have diacriticals or homoglyphs, the handle is stored with diacriticals and homoglyphs normalized.
-So `Zoë.35` and `Zoe.35` will both resolve to the same MSA Id.
```
Comment (jeanettedepatie) [inline pallets/handles/README.md:20]: Thus ... would.

**Review by jeanettedepatie — COMMENTED** [review]
Specific suggestions offered.  But okay as is.

File: pallets/handles/README.md (hunk 2 above)
Comment (mattheworris) [inline pallets/handles/README.md:16]:
```suggestion
A unique numeric suffix is appended to the base handle to form the display handle, allowing multiple users to select the same base handle.
```

**Review by mattheworris — APPROVED** [review]

File: pallets/handles/README.md (hunk 1 above)
Comment (wilwade) [reply pallets/handles/README.md:11]:
```suggestion
- **Suffix:** The suffix is a numeric value appended to the user's base handle to ensure the display handle (base handle + suffix) is unique.
```

File: pallets/handles/README.md (hunk 2 above)
Comment (wilwade) [reply pallets/handles/README.md:16]:
```suggestion
In order to allow multiple users to select the same base handle, a unique numeric suffix is appended to the Base Handle to form the Display Handle.
```

File: pallets/handles/README.md (hunk 3 above)
Comment (wilwade) [reply pallets/handles/README.md:18]:
```suggestion
For example, if there are two users who choose the handle `alice`, one would be `alice.57` and the other `alice.84`.
```

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**